synapse/tests/test_lib_certdir.py

```diff
@@ -39,13 +39,17 @@ def test_certdir_user(self):
     def test_certdir_host(self):
         with self.getCertDir() as cdir:
             cdir.genCaCert('syntest')
-            cdir.genHostCert('visi.vertex.link', signas='syntest')
+            cdir.genHostCert('visi.vertex.link', signas='syntest', sans='DNS:vertex.link,DNS:visi.vertex.link,DNS:vertex.link')
```

Comment:
My comment was removed when I pushed. I added `DNS:vertex.link` to `sans` twice here and have the test validating the SANs from the loaded cert now (and making sure that `vertex.link` only shows up once).